src/classes/ADDR_Contact_TDTM.cls

```diff
@@ -29,14 +29,15 @@
 */
 /**
 * @author Salesforce.org
-* @date 2014
+* @date 2016
 * @group Addresses
 * @group-content ../../ApexDocContent/Addresses.htm
 * @description Trigger Handler on Contact for Address management. 
 */
 public class ADDR_Contact_TDTM extends TDTM_Runnable {
```

Comment:
* Classes must use `with sharing` to enforce record level access and avoid bypassing record level security
* Classes and triggers must use the `UpperCamelCase` naming convention.